modules/linear_boltzmann_solvers/a_lbs_solver/lbs_solver.h

```diff
@@ -1,15 +1,15 @@
 #pragma once
 
-#include "framework/physics/SolverBase/chi_solver.h"
+#include "framework/physics/solver_base/solver.h"
 
-#include "modules/LinearBoltzmannSolvers/A_LBSSolver/Groupset/lbs_groupset.h"
-#include "framework/math/SpatialDiscretization/SpatialDiscretization.h"
-#include "framework/math/LinearSolver/linear_solver.h"
-#include "modules/LinearBoltzmannSolvers/A_LBSSolver/lbs_structs.h"
-#include "framework/mesh/SweepUtilities/sweep_namespace.h"
-#include "framework/mesh/SweepUtilities/SweepBoundary/sweep_boundary.h"
+#include "modules/linear_boltzmann_solvers/a_lbs_solver/groupset/lbs_groupset.h"
```

Comment:
some weird looking things are showing up like this a_ but I think this will be addressed through another process later.

Reply:
Yeah, the dir names under `modules/linear_boltzmann_solvers` start with a letter. It looks like somebody was trying to create some sort of sorting 🙃 . I kept it for now, but IMO that should go away...